core/tests/puppeteer-acceptance-tests/utilities/user/blog-post-editor.ts

```diff
@@ -145,22 +149,73 @@ export class BlogPostEditor extends BaseUser {
   }
 
   /**
+   * This is a composite function that can be used when a straightforward, simple blog published is required.
    * This function publishes a blog post with given title.
    */
   async publishNewBlogPostWithTitle(newBlogPostTitle: string): Promise<void> {
-    await this.addUserBioInBlogDashboard();
-    await this.clickOn(LABEL_FOR_NEW_BLOG_POST_CREATE_BUTTON);
+    await this.navigateToBlogEditorPage();
+    await this.uploadBlogPostThumbnailImage();
     await this.expectPublishButtonToBeDisabled();
 
-    await this.uploadBlogPostThumbnailImage();
+    await this.updateTitleTo(newBlogPostTitle);
+    await this.updateBodyTextTo('test blog post body content');
+    await this.selectTags('News', 'International');
+    await this.saveTheChanges();
+
+    await this.publishTheBlogPost();
+  }
+
+  /**
+   * This function navigates to the blog editor page.
+   */
+  async navigateToBlogEditorPage(): Promise<void> {
+    await this.addUserBioInBlogDashboard();
+    await this.clickOn(LABEL_FOR_NEW_BLOG_POST_CREATE_BUTTON);
     await this.expectPublishButtonToBeDisabled();
+  }
 
+  /**
+   * This function updates the title of the blog post.
+   */
+  async updateTitleTo(newBlogPostTitle: string): Promise<void> {
     await this.type(blogTitleInput, newBlogPostTitle);
     await this.page.keyboard.press('Tab');
-    await this.type(blogBodyInput, 'test blog post body content');
+  }
+
+  /**
+   * This function updates the body text of the blog post.
+   */
+  async updateBodyTextTo(newBodyText: string): Promise<void> {
+    await this.type(blogBodyInput, newBodyText);
+  }
+
+  /**
+   * This function saves the blog post.
+   */
+  async saveTheChanges(): Promise<void> {
     await this.clickOn('button.mat-button-toggle-button');
```

Comment:
Please store this as a constant. Ditto for others.

Reply:
Done